### ISSUE TITLE
disableStrictSSL with http

### ISSUE THREAD

**Issue by rxxy** [acacode/swagger-typescript-api#453]
If you use both http protocol and disableStrictSSL:true, it will cause a request error.

The disableStrictSSL option should be applied only in the case of https

![image](https://user-images.githubusercontent.com/14834156/201515062-8d57bd7c-4bde-4811-86e5-3bd93d05201f.png)

This is a small optimization

**Comment by js2me** [comment]
Hi @rxxy  !   
As hot fix, you can add this to your codegen config   

```
const http = require('http')

let config = {
   requestOptions: {
      agent: new http.Agent({
         rejectUnauthorized: false,
      }),
   }
}
```

**Comment by js2me** [comment]
@rxxy also I suggest you to use path (`require("path")`) for resolving paths for output and templates   
![image](https://user-images.githubusercontent.com/16340911/201532792-b7c2c006-3224-4d8d-88bb-5fdbf0dee3dd.png)

**Comment by js2me** [comment]
@rxxy should be fixed in 12.0.1
